lib/reline.rb

```diff
@@ -610,6 +610,10 @@ def self.core
       core.filename_quote_characters = ""
       core.special_prefixes = ""
       core.add_dialog_proc(:autocomplete, Reline::DEFAULT_DIALOG_PROC_AUTOCOMPLETE, Reline::DEFAULT_DIALOG_CONTEXT)
+      core.dialog_default_bg_color = 46 # Cyan
+      core.dialog_default_fg_color = 37 # White
+      core.dialog_pointer_bg_color = 45 # Maggenta
```

Comment:
Maggenta => Magenta

Reply:
@Maumagnaguagno Thanks. I have fixed it.